libs/parser-sql/src/postgresParser.ts

```
@@ -0,0 +1,1652 @@
+    TransactionModeAst,
+    TypeColumnAst,
+    UnionClauseAst,
+    UpdateColumnAst,
+    UpdateStatementAst,
+    WhereClauseAst,
+    WindowClauseAst,
+    WindowClauseContentAst
+} from "./postgresAst";
+
+const LineComment = createToken({name: 'LineComment', pattern: /--.*/, group: 'comments'})
+const BlockComment = createToken({name: 'BlockComment', pattern: /\/\*[^]*?\*\//, line_breaks: true, group: 'comments'})
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)